cms/models.py

```diff
@@ -953,7 +954,7 @@ def bootcamp_certificate(
             certificate = BootcampRunCertificate.objects.get(
                 uuid=uuid, is_revoked=False
             )
-        except BootcampRunCertificate.DoesNotExist:
+        except (BootcampRunCertificate.DoesNotExist, ValidationError):
```

Comment:
Do we need this? When and how is this Validation error going to be thrown? Even if this error is thrown we should let the program break instead of a 404.